### PR TITLE
Create seed data for the Celebrities topic of the topics table

### PR DESCRIPTION
**Pull request by monicamow**
- creates one topic object (Celebrities)
- creates twitter_handle objects which includes one real twitter_handle and one plus fake twitter_handle(s)
- this file is required by seeds.rb, which gets run in a rake task to populate our database

this pull requests will add a Topic object for Celebrities and multiple TwitterHandle objects owned by certain celebrities to the seeds.rb file which will be run in a rake task

- added one real account and one parody account for: Ellen, Elon Musk, Queen Elizabeth, David Karp, Taylor Swift, Kanye West

this is an update to PR #15

### CODE REVIEW

File: db/celebs_seeds.rb [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+#######################
+# ELLEN DEGENERES
+# Real
+real_ellen_degeneres = TwitterHandle.create!(
+twitter_handle: "theellenshow",
+topic: topic)
+
+# Fake
+fake_ellen_degeneres = TwitterHandle.create!(
```
Comment (monicamow) [inline db/celebs_seeds.rb]: can remove local variables for fake accounts to improve speed of rake task

File: db/celebs_seeds.rb [hunk 2]
```diff
@@ -0,0 +1,83 @@
+topic = Topic.create!(
+topic: "Celebrities")
+
+#######################
+# ELLEN DEGENERES
+# Real
+real_ellen_degeneres = TwitterHandle.create!(
+twitter_handle: "theellenshow",
+topic: topic)
```
Comment (tjukes) [inline db/celebs_seeds.rb]: This needs to be `topic_id: topic.id` for every entry. It's a reference to the ID of the entry in the Topics table (integer), not to the entry itself -- see schema.rb.

Comment (cody-code-wy) [inline db/celebs_seeds.rb]: @tjukes no the current one is correct by AR conventions.

**Comment by cody-code-wy** [comment]
I like the comments in the code, very nice style! +1

**Pull request merged**